stock_release_channel_shipment_advice_process_end_time/models/stock_release_channel.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from datetime import timedelta
+
+from odoo import api, fields, models
+
+
+class StockReleaseChannel(models.Model):
+
+    _inherit = "stock.release.channel"
+    shipment_advice_arrival_delay = fields.Integer(
+        compute="_compute_shipment_advice_arrival_delay",
+        store=True,
+        readonly=False,
+        help="The delay between the release channel process end time and the arrival "
+        "of shipments to the dock.",
+    )
+
+    @api.depends("warehouse_id")
+    def _compute_shipment_advice_arrival_delay(self):
+        for rec in self:
+            if not rec.shipment_advice_arrival_delay:
+                rec.shipment_advice_arrival_delay = (
+                    rec.warehouse_id.release_channel_shipment_advice_arrival_delay
+                )
+
+    def _get_shipment_advice_arrival_delay(self, warehouse):
+        self.ensure_one()
+        if self.shipment_advice_arrival_delay:
+            return self.shipment_advice_arrival_delay
+        if not warehouse:
+            return 0
+        return warehouse.release_channel_shipment_advice_arrival_delay
```

Comment:
As I said before, I would drop this because shipment_advice_arrival_delay is computed store that already defaults on warehouse.release_channel_shipment_advice_arrival_delay.
So if the value was set intentionally to 0 on shipment_advice_arrival_delay, you have to use that value, and not the default value